### PR TITLE
Add Deepavali from 2022 to 2030 for Singapore

### PR DESCRIPTION
**Pull request by hprobotic**
Add holidays from 2022 to 2030 for:
- Singapore (https://holidays.kayaposoft.com/public_holidays.php?year=2022&country=sgp&region=#)

- [x] Tests with a significant number of years to be tested for your calendar.
- [x] Changelog amended with a mention describing your changes

### CODE REVIEW

**Comment by jack-pace** [comment]
Excellent - I was just about to make a PR adding these dates when I found one already existed!
Would be grateful of a speedy merge :smile:

File: workalendar/asia/singapore.py [hunk 1]
```diff
@@ -53,7 +54,16 @@ class Singapore(WesternMixin, IslamicMixin, ChineseNewYearCalendar):
         2018: date(2018, 11, 6),
         2019: date(2019, 10, 27),
         2020: date(2020, 11, 14),
-        2021: date(2021, 11, 4),  # This might change
+        2021: date(2021, 11, 4),
+        2022: date(2022, 10, 24),
+        2023: date(2023, 12, 11),
```
Comment (jack-pace) [inline workalendar/asia/singapore.py:59]: Should this be `date(2023, 11, 12)`? Or even `date(2023, 11, 13)`?
The website you reference says that the public holiday is on the 13th November 2023 in lieu of the 12th.

**Comment by brunobord** [comment]
thank you for this very valuable PR.

Along with answering the question by @jack-pace - and eventually assign the right date, could you please make sure that your code is flake8-ready? There's only one error, so this shouldn't be too hard:

> workalendar/asia/singapore.py:34:80: E501 line too long (99 > 79 characters)

File: workalendar/asia/singapore.py (hunk 1 above)
Comment (brunobord) [reply workalendar/asia/singapore.py:59]: I agree with @jack-pace.

```suggestion
        2023: date(2023, 11, 13),
```

File: workalendar/asia/singapore.py [hunk 2]
```diff
@@ -31,6 +31,7 @@ class Singapore(WesternMixin, IslamicMixin, ChineseNewYearCalendar):
 
     # Diwali/Deepavali is sometimes celebrated on a different day to India
     # so this can't be put into a HinduMixin
+    # Reference: https://holidays.kayaposoft.com/public_holidays.php?year=2022&country=sgp&region=#
```
Comment (brunobord) [inline workalendar/asia/singapore.py:34]:
```suggestion
    # Ref:
    # https://holidays.kayaposoft.com/public_holidays.php?year=2022&country=sgp
```

**Comment by brunobord** [comment]
Wow, I didn't know I could commit directly on the PR without having to make my own branch. a WHOLE NEW WORLD IS AFOOT

**Comment by brunobord** [comment]
@hprobotic and @jack-pace the latest release of Workalendar (14.2.0) embeds your change. Happy upgrading!

* https://github.com/peopledoc/workalendar/releases/tag/14.2.0
* https://pypi.org/project/workalendar/14.2.0/

**Comment by hprobotic** [comment]
Hey thank @brunobord  for your help on this. I have been hospital for stomach surgery recently, didn't check GitHub notifications for a week

**Pull request merged**